docs/HowToFAQ/FAQNewline.md

```diff
@@ -0,0 +1,13 @@
+---
+title: It looks like, when compiling to C#, my print statements don't show up if I don't have \n at the end of the string.
+---
+
+## Question
+
+It looks like, when compiling to C#, my print statements don't show up if I don't have \n at the end of the string.
+
+## Answer
+
+This is likely to be platform-dependent problem; the problem is whether the I/O system for the target platform does a flush of the output streams prior to terminating a program. The workaround is for the user program to do a `print "\n";` at the end of the program.
```

Comment:
Is there any platform on which pipes aren't flushed upon exit?

Reply:
Tested and fixed the answer